Api/Controllers/MigratorController.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Application.Migrator;
+using Microsoft.AspNetCore.Mvc;
+using System.Text;
+
+namespace Api.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class MigratorController : Controller
+    {
+        private readonly SQLToMongoMigrator _sQLToMongoMigrator;
+        public MigratorController(SQLToMongoMigrator sQLToMongoMigrator)
+        {
+            _sQLToMongoMigrator = sQLToMongoMigrator;
+        }
+        [HttpGet("MigrateGeoLocation")]
+        public IActionResult MigrateGeoLocation()
+        {
+           StringBuilder results = new StringBuilder();
+           results.AppendLine(_sQLToMongoMigrator.MigrateGeolocation());
```

Comment:
Why did you decide to use StringBuilder here? As far as I know, the benefits of using it only materialize in a situation where you're concatenating multiple strings.

Reply:
I plan to use one end point for all of them so will build more strings eventually.